### PR TITLE
📝🎨update enum member documentation to sphinx compatible format

### PR DESCRIPTION
**Pull request by hf-kklein**
Repeates the steps from #78 for all enum (member)s

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ In combination with this substitution:
 
 .. code-block:: Shell
 
-    "$wert": "$wert", # $bedeutung
+    "$wert": "$wert", #: $bedeutung
```
Comment (hf-krechan) [inline README.rst:63]: What is the purpose of the `:`?

File: src/bo4e/enum/themengebiet.py [hunk 2]
```diff
@@ -8,64 +8,64 @@ class Themengebiet(StrEnum):
...
+    ALLGEMEINER_INFORMATIONSAUSTAUSCH = "ALLGEMEINER_INFORMATIONSAUSTAUSCH"  #: Allgemeiner Informationsaustausch
+    AN_UND_ABMELDUNG = "AN_UND_ABMELDUNG"  #: An- und Abmeldung
+    ANSPRECHPARTNER_ALLGEMEIN = "ANSPRECHPARTNER_ALLGEMEIN"  #: Ansprechpartner Allgemein
+    ANSPRECHPARTNER_BDEW_DVGW = "ANSPRECHPARTNER_BDEW_DVGW"  #: Ansprechpartner BDEW/DVGW
+    ANSPRECHPARTNER_IT_TECHNIK = "ANSPRECHPARTNER_IT_TECHNIK"  #: Ansprechpartner IT/Technik
+    BILANZIERUNG = "BILANZIERUNG"  #: Bilanzierung
+    BILANZKREISKOORDINATOR = "BILANZKREISKOORDINATOR"  #: Bilanzkreiskoordinator
+    BILANZKREISVERANTWORTLICHER = "BILANZKREISVERANTWORTLICHER"  #: Bilanzkreisverantwortlicher
     DATENFORMATE_ZERTIFIKATE_VERSCHLUESSELUNGEN = (
         "DATENFORMATE_ZERTIFIKATE_VERSCHLUESSELUNGEN"  # Datenformate, Zertifikate, Verschlüsselungen
```
Comment (hf-krechan) [inline src/bo4e/enum/themengebiet.py:20]: The `:` is here missing, isn't it?

**Review by hf-krechan — CHANGES_REQUESTED** [review]
Just the question what is the purpose of `:` and if it's important, one enum entry is missing it.

**Comment by hf-kklein** [comment]
> Just the question what is the purpose of `:`

see #78 it's described there including minimal examples (tl;dr: doc entries for enum members with trailing `#:` do have a sphinx docstring, without don't)

File: src/bo4e/enum/themengebiet.py (hunk 2 above)
Comment (hf-kklein) [reply src/bo4e/enum/themengebiet.py:20]: probably my regex search and replace failed for those entries... maybe because they span multiple lines.

Comment (hf-kklein) [reply src/bo4e/enum/themengebiet.py]: https://github.com/Hochfrequenz/BO4E-python/pull/81/commits/143447900d39e11946b4587c2c513402727548c5

**Review by hf-krechan — APPROVED** [review]
Go go go

**Pull request merged**